nubis/puppet/salesforce_sfmc.pp

```diff
@@ -15,7 +15,7 @@
 python::pip { 'sfmc-fetcher':
   ensure     => 'present',
   virtualenv => "${virtualenv_path}/data-integrations",
-  #url        => 'git+https://github.com/mozilla-it/data-integrations@2126a2098c28e3fe447305c2438bd18f4fefa7a1',
+  url        => 'git+https://github.com/mozilla-it/data-integrations@2126a2098c28e3fe447305c2438bd18f4fefa7a1',
```

Comment:
This repo isn't pip installable, it's missing a setup.py at a minimum to drive installation.
```
$> pip install --user git+https://github.com/mozilla-it/data-integrations@2126a2098c28e3fe447305c2438bd18f4fefa7a1
Collecting git+https://github.com/mozilla-it/data-integrations@2126a2098c28e3fe447305c2438bd18f4fefa7a1
  Cloning https://github.com/mozilla-it/data-integrations (to 2126a2098c28e3fe447305c2438bd18f4fefa7a1) to /tmp/pip-4Bg4wP-build
  Could not find a tag or branch '2126a2098c28e3fe447305c2438bd18f4fefa7a1', assuming commit.
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    IOError: [Errno 2] No such file or directory: '/tmp/pip-4Bg4wP-build/setup.py'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-4Bg4wP-build/
```